### PR TITLE
issue #284 - ability to hide Thead in a Table

### PR DESCRIPTION
**Pull request by cklab**
- add handling for a hideTableHeader property for Table

### CODE REVIEW

File: src/reactable/table.jsx [hunk 1]
```diff
@@ -477,24 +478,28 @@ export class Table extends React.Component {
 
         let noDataText = this.props.noDataText ? <tr className="reactable-no-data"><td colSpan={columns.length}>{this.props.noDataText}</td></tr> : null;
 
-        return <table {...props}>
-            {columns && columns.length > 0 ?
-             <Thead columns={columns}
-                 filtering={filtering}
-                 onFilter={filter => {
+        var tableHeader;
```
Comment (glittershark) [inline src/reactable/table.jsx]: let's explicitly initialize this to `null`.

File: tests/reactable_test.jsx [hunk 2]
```diff
@@ -657,6 +657,46 @@ describe('Reactable', function() {
...
+                ReactableTestUtils.expectRowText(0, ['Griffin Smith', '18', '']);
+            });
+
+            it('renders the second row with the correct data', function() {
+                ReactableTestUtils.expectRowText(1, ['Lee Salminen', '23', '']);
+            });
+
+            it('renders the third row with the correct data', function() {
+                ReactableTestUtils.expectRowText(2, ['', '28', 'Developer']);
+            });
```
Comment (glittershark) [inline tests/reactable_test.jsx]: I feel like these first four `it` blocks are kinda redundant, tbh

Comment (cklab) [inline tests/reactable_test.jsx]: I didn't have them at first but I think they're beneficial to have. Granted maybe not all four, but at least the table & the first row. 

What good is the thead not existing if the entire table doesn't exist? I wanted to make sure that hiding the header didn't inadvertently affect the table being rendered, in this change & any future change to the code.

Comment (glittershark) [inline tests/reactable_test.jsx]: oh, yeah, I feel that. You've convinced me, this is :ok_hand:

Comment (cklab) [inline tests/reactable_test.jsx]: back in they go... haha

**Comment by glittershark** [comment]
please squash all your commits into one and this is good to go

**Comment by cklab** [comment]
rebase again after the most recent pull request

**Comment by glittershark** [comment]
Thanks for the PR!

**Pull request merged**